mobile/cmp/popover/Popover.scss

```diff
@@ -0,0 +1,27 @@
+.xh-popover {
+  &__content-wrapper {
+    position: fixed;
+    z-index: 21;
+    background-color: var(--xh-bg);
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0.3);
+  }
+
+  &__content-overlay {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    z-index: 10;
+    background-color: transparent;
+
+    &--mask {
+      background-color: var(--xh-mask-bg);
```

Comment:
If we did update the prop to `withBackdrop` or similar, we could add a distinct `--xh-backdrop-bg` var.

For a client project where we just added a usage of the Blueprint drawer, I found that I wanted distinct shading for the backdrop, which I set to be darker so as to better mask and then set off the modal content in front of it, vs the mask, which I wanted to be lighter so it was less jarring when rendered over e.g. an existing grid full of data that was simply refreshing.

I can show you if helpful in practice - I hadn't appreciated the distinction until just the other day